tests/test-nginx/cases/ngx_http_upstream_check_module/http_check.t

```diff
@@ -538,3 +538,59 @@ GET /
 --- request
 GET /
 --- response_body_like: ^<(.*)>$
+
+=== TEST 19: the http_check with type!=http and check_http_send configured
+--- http_config
+    upstream test{
+        server 127.0.0.1:1970;
+        check_keepalive_requests 10;
+        check interval=3000 rise=1 fall=1 timeout=1000 type=tcp;
+        check_http_send "GET / HTTP/1.0\r\nConnection: keep-alive\r\n\r\n";
+        check_http_expect_alive http_2xx http_3xx;
+    }
+
+    server {
+        listen 1970;
+
+        location / {
+            root   html;
+            index  index.html index.htm;
+        }
+    }
+
+--- config
+    location / {
+        proxy_pass http://test;
+    }
+
+--- nginx config test
```

Comment:
I didn't find documentation about this section, and your case can not passed.

Can you tell me how to run your test case, including the version of Test::Nginx, and your command.